site/oauth2-examples-oauth0.md

```diff
@@ -0,0 +1,134 @@
+# Use https://auth0.com/ as OAuth 2.0 server
+
+We are going to test 3 OAuth flows:
+1. Access management ui via a browser
+2. Access management rest api
+3. Access AMQP protocol
+
```

Comment:
Same as previous comments above, avoid using "We" in documentation - For the above, use "Let's test 3 OAuth flows:"